network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -72,20 +95,16 @@ class PublicationPage(FoundationMetadataPageMixin, Page):
     )
 
     content_panels = Page.content_panels + [
-        FieldPanel('subtitle'),
-        FieldPanel('secondary_subtitle'),
-        FieldPanel('publication_date'),
-        ImageChooserPanel('hero_image'),
-        DocumentChooserPanel('publication_file'),
+        MultiFieldPanel([
```

Comment:
minor quibbles:
* if Authors are optional and therefore not always entered, I'd rather have them below other required or commonly entered stuff like the Hero
* curious about the motivation to put the other hero fields into a multifield panel, im assuming thats to group things a bit more semantically. In which case wouldn't authors be apart of that since they are also in the hero?

Reply:
No reason. It's because I copy/pastas'd from some other authors work. Will move `authors` into the hero `MultiFieldPanel`